Makefile

```diff
@@ -117,7 +119,8 @@ cmd/bin/subctl-%: $(shell find . -name "*.go") $(VENDOR_MODULES)
 	export GOARCH GOOS; \
 	$(SCRIPTS_DIR)/compile.sh \
 		--ldflags "-X 'github.com/submariner-io/subctl/pkg/version.Version=$(VERSION)' \
-		       -X 'github.com/submariner-io/submariner-operator/api/submariner/v1alpha1.DefaultSubmarinerOperatorVersion=$${DEFAULT_IMAGE_VERSION#v}'" \
+		       -X 'github.com/submariner-io/submariner-operator/api/submariner/v1alpha1.DefaultSubmarinerOperatorVersion=$${DEFAULT_IMAGE_VERSION#v}' \
+		       -X 'github.com/submariner-io/submariner-operator/api/submariner/v1alpha1.DefaultRepo=${DEFAULT_REPO}'" \
```

Comment:
`DEFAULT_REPO` needs to be surrounded by parentheses, not curly braces, see https://github.com/submariner-io/subctl/pull/172#discussion_r924263455

Reply:
Technically both should work according to the docs [1] but since we want to avoid confusion by devs it's probably best to use parenthesis to explicitly signal it's a make variable.

[1] https://www.gnu.org/software/make/manual/html_node/Reference.html#Reference